### ISSUE TITLE
对测试集数据的要求是什么，我用其他的图像测试出来都是一个蓝块，没有结果

### ISSUE THREAD

**Issue by hantong263** [TachibanaYoshino/Remote-sensing-image-semantic-segmentation#7]

**Comment by TachibanaYoshino** [comment]
Same as the training dataset type

**Comment by Sun-Xinnnnn** [comment]
Me too!!!do u solve this problem？？

**Comment by Sun-Xinnnnn** [comment]
> Same as the training dataset type

How can other datasets be adjusted to the same format as the training set?

**Comment by TachibanaYoshino** [comment]
Of course, you can adjust the palette and one-hot preprocessing of the dataset.

**Comment by Sun-Xinnnnn** [comment]
> 当然，您可以调整数据集的调色板和 one-hot 预处理。

非常感谢您的回复！！！我直接跑的预测，用您的模型，发现是蓝色的，是因为one-hot 预处理不对吗？

**Comment by TachibanaYoshino** [comment]
You need to make sure that your data is of the same category as my predefined category, and the palette is the same. For rivers and lakes, the processed colored semantic regions are blue. Please note that the output result of the network is the predicted segmentation map, and then according to the color of the palette corresponding to the fixed category number, the map of different color semantics is generated.
